e2e/tests/cohorts/cohorts-create.spec.ts

```diff
@@ -121,7 +121,7 @@ describe('User can create new Cohorts', () => {
     const dataPage = new DataPage(page);
 
     // Save url
-    const workspaceDataUrl = await page.url();
+    const workspaceDataUrl = page.url();
```

Comment:
`page.url()` is also non-async call. removed `await` here and several other places.